wpilibc/src/main/native/cpp/Ultrasonic.cpp

```diff
@@ -165,7 +159,7 @@ void Ultrasonic::SetEnabled(bool enable) {
 void Ultrasonic::InitSendable(SendableBuilder& builder) {
   builder.SetSmartDashboardType("Ultrasonic");
   builder.AddDoubleProperty(
-      "Value", [=]() { return GetRangeInches(); }, nullptr);
+      "Value", [=] { return GetRange().to<double>(); }, nullptr);
```

Comment:
This changes the displayed value from inches to meters.  Recommend we do a units conversion to inches?